compiler/src/iree/compiler/Dialect/Util/Transforms/Passes.td

```diff
@@ -155,4 +155,33 @@ def TestFloatRangeAnalysis : Pass<"iree-util-test-float-range-analysis", ""> {
   }];
 }
 
+
+//===----------------------------------------------------------------------===//
```

Comment:
since the tracing passes are specific to tracing utils, we should just move them into there - we don't need to use tablegen for them and can just have C++ classes in TracingUtils.cpp - we want to keep this as contained as we can and they don't really belong in util as they are tracing infra instead of actual passes a user would ever add themselves